test-assets/role-manifests/bosh-run-bad-cpu.yml

```diff
@@ -5,3 +5,6 @@ roles:
   run:
     memory: 1
     virtual-cpus: -2
+configuration:
+  templates:
+    properties.tor.hostname: '((IP_ADDRESS))((DNS_RECORD_NAME))'
```

Comment:
We could of course (;))add another marker flag to stop that, but then it gets just silly.